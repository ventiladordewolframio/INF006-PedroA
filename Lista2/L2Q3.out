0 (0) 1 (1) 4 (0) 5 (2) 6 (0) 8 (1) 9 (0) 
0 (0) 2 (1) 
