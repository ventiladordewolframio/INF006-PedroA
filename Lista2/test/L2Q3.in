a 3 r 3 a 5 a 7 a 8 a 1 a 4 a 6 a 1 r 9 a 0 r 7
a 1 a 2 r 0 r 1