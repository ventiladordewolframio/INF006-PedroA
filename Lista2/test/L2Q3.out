0 (2) 1 (1) 1 (3) 4 (2) 5 (0) 6 (2) 8 (1) 9 (2)
0 (1) 2 (0)